pkg/api/v1beta3/types.go

```diff
@@ -603,6 +603,8 @@ type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty" description:"Maximum amount of compute resources allowed"`
 	// Requests describes the minimum amount of compute resources required.
+	// Note: 'Requests' are honored only for Persistent Volumes as of now.
+	// TODO: Update the scheduler to use 'Requests' instead of 'Limits' for the purposes of scheduling.
```

Comment:
Align the TODO with the resources.md document:

TODO: Update the scheduler to use `Requests` in addition to `Limits`.  "If Request is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value"